templates/decrypt-tls-assets

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+rkt run \
```

Comment:
For my interest, why are we using rocket here? Is there something we should write comments for? Reading through the code of the template I understand that it basically decrypts the encrypted keys. What happens with the decrypted keys? Do they just stay inside of the cert dir? I am not sure of I miss something. Having some comments in here would maybe help developers reading through the code the first time.